ols/src/ui/gradio_ui.py

```diff
@@ -14,8 +15,8 @@ class GradioUI:
 
     def __init__(
         self,
-        ols_url="http://127.0.0.1:8080/v1/query",
-        conversation_id=None,
+        ols_url: str = "http://127.0.0.1:8080/v1/query",
+        conversation_id: Optional[str] = None,
```

Comment:
I think we started standardizing on using the `str | None = None` syntax for this, which is the python 3.11 style?

Reply:
ok will change it.